File: All Codes/URI_1040.c
#include <stdio.h>
int main()
{
    double N1, N2, N3, N4, N5, a,b;
    scanf ("%lf%lf%lf%lf", &N1,&N2,&N3,&N4);
    a=((N1*2)+(N2*3)+(N3*4)+N4)/10;
    printf ("Media: %.1lf\n",a);
    if(a>=7)
        printf ("Aluno aprovado.\n");
    else if(a<5)
        printf("Aluno reprovado.\n");
    else if(a>=5&&a<=6.9)
    {
        printf ("Aluno em exame.\n");
        scanf("%lf", &N5);
        printf ("Nota do exame: %.1lf\n",N5);
        b=(N5+a)/2;
        if(b>=5)
            printf ("Aluno aprovado.\n");
        else if(b<=4.9)
            printf ("Aluno reprovado.\n");
        printf ("Media final: %.1lf\n",b);
    }
    return 0;
}

File: All Codes/URI_1040.py
a = [float(x) for x in input().split()]

m = ( (a[0] * 2) + (a[1] * 3) + (a[2] * 4) + a[3] ) / 10
print(f"Media: {m:.1f}")

if m >= 7:
    print("Aluno aprovado.")
elif (m >= 5 and m <= 6.9):
    print("Aluno em exame.")
    a.append(float(input()))
    print(f"Nota do exame: {a[-1]:.1f}")
    m = (m + a[-1]) / 2
    if m >= 5:
        print("Aluno aprovado.")
    elif m <= 4.9:
        print("Aluno reprovado.")
    print(f"Media final: {m:.1f}")
else:
    print("Aluno reprovado.")
